src/Facebook/FacebookRedirectLoginHelper.php

```diff
@@ -188,8 +188,17 @@ public function getSessionFromRedirect()
         '/oauth/access_token',
         $params
       ))->execute()->getResponse();
-      if (isset($response['access_token'])) {
-        return new FacebookSession($response['access_token']);
+
+      // Graph v2.3 and greater return objects on the /oauth/access_token endpoint
+      $accessToken = null;
+      if ($response instanceof \stdClass && isset($response->access_token)) {
```

Comment:
Maybe you can also use `property_exists($response, 'access_token')`, maybe, I'm +0 about that haha